.travis.yml

```diff
@@ -10,13 +10,9 @@ before_install:
 
 install:
   - pip install -r src/requirements.txt
-  - pip install fakeRPiGPIO
+  - pip install -r src/dev-requirements.txt
   - yes | pip install git+https://github.com/duxingkei33/orangepi_PC_gpio_pyH3.git -v
-  - pip install git+https://github.com/oaubert/python-vlc.git
-  - pip install pylint
   - cp src/config.template.yaml src/config.yaml
 
-script: 
-  - pylint --errors-only --extension-pkg-whitelist=alsaaudio src/main.py
-  - pylint --errors-only --extension-pkg-whitelist=alsaaudio,pyA20 src/alexapi
-  - pylint --errors-only src/auth_web.py
+script:
+  - pylint --rcfile=pylintrc --extension-pkg-whitelist=alsaaudio,pyA20 src/*.py
```

Comment:
If I'm correct:
- we don't need `--extension-pkg-whitelist=alsaaudio,pyA20` here anymore, since it's in the pylintrc file
- this doesn't check the `src/alexapi/` directory